R/global.R

```diff
@@ -14,5 +14,7 @@ session <- local({
 
 .onLoad <- function(...) {
   options(error = traceback)
+  options(vsc.previewPromises = FALSE)
```

Comment:
The usage here may not be desired because it overwrites any user value on package load.

For example, if user puts `options(vsc.previewPromises = TRUE)` in `.Rprofile`, it will not work becuase it its overwritten to `options(vsc.previewPromises = FALSE)` on package load.